test/scenarios/driver/vagrant/molecule/default/tests/test_default.py

```diff
@@ -7,7 +7,7 @@
 
 
 def test_hostname(host):
-    assert 'instance' == host.check_output('hostname -s')
+    assert host.check_output('hostname -s') == 'instance'
```

Comment:
Why is this flipped?  Looks unrelated and our pattern is `expected == returned`.

Reply:
This was actually the only test case which was written as `expected == returned` and my linter complained that test cases should actually be `returned == expected`. But I can revert it back, if it bothers you.